example_pipeline/04_run_state_annotation.py

```diff
@@ -114,7 +117,11 @@ def _get_score(vec):
         predicted_state_vector = annotator.predict(signal_array)
         predicted_states, predicted_intervals = convert_state_vector_to_state_intervals(
             predicted_state_vector, mapping=int_to_state, time_resolution=time_resolution)
-        export_hypnogram(dataset['file_path_automated_state_annotation'], predicted_states, predicted_intervals)
+        
+        with EdfReader(dataset['file_path_raw_signals']) as f:
+            edf_header = f.getHeader() # ?read header from the original file
+ 
+        export_hypnogram(dataset['file_path_automated_state_annotation'], predicted_states, predicted_intervals, edf_header)
```

Comment:
This behaviour is implemented in c020140ce4692c482b6461f92aae7aeeb99c31e4. However, the headers are only copied, if the target file for the annotations is indeed in an EDF format.